application/controllers/service_helper.py

```diff
@@ -99,19 +31,26 @@ def service_request(service, endpoint, search_value):
     auth = (current_app.config.get("SECURITY_USER_NAME"), current_app.config.get("SECURITY_USER_PASSWORD"))
 
     try:
-        service = {
+        service_root = {
             "survey-service": current_app.config["SURVEY_URL"],
             "collectionexercise-service": current_app.config["COLLECTION_EXERCISE_URL"],
             "case-service": current_app.config["CASE_URL"],
             "party-service": current_app.config["PARTY_URL"],
         }[service]
-        service_url = f"{service}/{endpoint}/{search_value}"
+        service_url = f"{service_root}/{endpoint}/{search_value}"
         log.info(f"Making request to {service_url}")
     except KeyError:
         raise RasError(f"service '{service}' not configured", 500)
 
-    response = requests.get(service_url, auth=auth)
-    response.raise_for_status()
+    try:
+        response = requests.get(service_url, auth=auth)
+        response.raise_for_status()
+    except requests.HTTPError:
+        raise RasError(f"{service} returned a HTTPError")
+    except requests.ConnectionError:
+        raise ServiceUnavailableException(f"{service} returned a connection error", 503)
+    except requests.Timeout:
+        raise ServiceUnavailableException(f"{service} has timed out", 504)
```

Comment:
Just good practise. I had to change how survey service returned, so update the calls (with tests)